src/python/pants/backend/python/subsystems/pex_build_util.py

```diff
@@ -236,21 +239,40 @@ def add_resolved_requirements(self, reqs, platforms=None):
     :param log: Use this logger.
     :param platforms: A list of :class:`Platform`s to resolve requirements for.
                       Defaults to the platforms specified by PythonSetup.
+    :param bool override_ptex_skip: When this PexBuilderWrapper is configured with
```

Comment:
Maybe more verbosely:
`override_ptex_build_do_actually_add_distribution`

Skip generally suggests to me that we're going to _not_ do something, rather than that we're going to skip not doing something :)

Reply:
Thank you!